src/styles/components/Breadcrumb.scss

```diff
@@ -12,41 +12,88 @@
  @import "../common/fonts";
  @import "../common/mixins";
  
+.breadcrumbs-container {
+  margin: 24px 0;
+}
  .breadcrumb {
    display: flex;
    flex-wrap: wrap;
    width: fit-content;
-   padding: 0 0;
+   padding: 0 2px;
    margin-bottom: 1rem;
```

Comment:
Lets keep things consistent.  Fonts should be in REMs only
padding, margin - REMs preferred, but ok if we want px too.  But keep it consistent in the page.

Reply:
Updated to rem